### PR TITLE
Add ESG to sponsors

### PR DESCRIPTION
**Pull request by bfbachmann**
Preview: https://5c09bc09f6d5ea2153f45b42--ubclaunchpad.netlify.com/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -608,6 +608,9 @@ <h1>Sponsors</h1>
     <a href="https://pullreminders.com/">
       <div class="sponsor" id="pull-reminders"></div>
     </a>
+    <a href="https://www.eastsidegames.com/">
+      <div class="sponsor" id="east-side-games"></div>
```
Comment (bobheadxi) [inline index.html]: are we not ordering these by amount contributed? if so they should be near the top after GitHub

Comment (frostyshadows) [reply index.html]: ^was just thinking the same thing

Comment (bfbachmann) [reply index.html]: Good call, done.

**Review by bobheadxi — APPROVED** [review]
should send an email to ESG thanking them, confirming we received the money, and let them know we've updated our website and look forward to working with them etc etc 👍

**Comment by bfbachmann** [comment]
@bobheadxi Already done, I emailed them 30 min ago. I didn't say anything about the website, though.

**Comment by bobheadxi** [comment]
Ticket needs to be updated then. https://github.com/ubclaunchpad/strategy/issues/20 @bfbachmann

**Pull request merged**